task.ps1

```diff
@@ -56,4 +58,12 @@ $Params = @{
 }
 Set-AzVMExtension @Params
 
-# Install Azure Monitor Agent VM extention -> 
+# Install Azure Monitor Agent VM extention ->
+Set-AzVMExtension `
+    -ResourceGroupName $resourceGroupName `
+    -VMName $vmName `
+    -Location $location `
+    -Name "AzureMonitorLinuxAgent" `
+    -Publisher "Microsoft.Azure.Monitor" `
+    -ExtensionType "AzureMonitorLinuxAgent" `
+    -TypeHandlerVersion "1.1"
```

Comment:
Ensure that the Azure Monitor Agent extension name and type are correct. Verify the latest version and correct naming convention for the extension in the Azure documentation.